dashboards-reports/server/index.ts

```diff
@@ -28,17 +28,18 @@ import {
   HttpServerInfo,
   PluginInitializerContext,
 } from '../../../src/core/server';
+import { ReportingConfigType } from './config';
```

Comment:
where is this `./config`?

Reply:
`git add .` somehow doesn't pick up everything. I added a new folder `server/config`, just updated the PR